testing/btest/Baseline/scripts.base.protocols.redis.pubsub/redis.log

```diff
@@ -0,0 +1,13 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+#separator \x09
+#set_separator	,
+#empty_field	(empty)
+#unset_field	-
+#path	redis
+#open XXXX-XX-XX-XX-XX-XX
+#fields	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	cmd.name	cmd.key	cmd.value	success	reply.value
+#types	time	string	addr	port	addr	port	string	string	string	bool	string
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	127.0.0.1	56162	127.0.0.1	6379	SUBSCRIBE	-	-	T	-
+XXXXXXXXXX.XXXXXX	ClEkJM2Vm5giqnMf4h	127.0.0.1	56163	127.0.0.1	6379	PUBLISH	-	-	T	-
+XXXXXXXXXX.XXXXXX	CHhAvVGS1DHFjwGM9	127.0.0.1	56162	127.0.0.1	6379	-	-	-	T	-
```

Comment:
For a follow-up, what's going on here?

Reply:
an array `message` from the server on the subscriber's connection

```
@data 127.0.0.1-56162-127.0.0.1-6379-tcp-resp 48
*3
$7
message
$10
my_channel
$8
hello :)
```

It's an edge case that I... haven't quite looked into how it needs to be handled :) It might just be best to immediately log them, I don't think they should be correlated with `CLIENT REPLY` stuff and it'll break that. If that's blocking I can hold